x/sp/keeper/msg_server.go

```diff
@@ -154,9 +167,9 @@ func (k msgServer) CreateStorageProvider(goCtx context.Context, msg *types.MsgCr
 func (k msgServer) EditStorageProvider(goCtx context.Context, msg *types.MsgEditStorageProvider) (*types.MsgEditStorageProviderResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
-	spAcc := sdk.MustAccAddressFromHex(msg.SpAddress)
```

Comment:
Do we allow to edit operator address for rotation? Shall we use Id when editing sp?

Reply:
Support it later.